components/script/script_thread.rs

```diff
@@ -570,6 +574,14 @@ impl ScriptThreadFactory for ScriptThread {
 }
 
 impl ScriptThread {
+    pub fn set_mutation_observer(observer: &MutationObserver) {
```

Comment:
I have changed this name, but I am not able to figure out how exactly do I reference this from the mutationobserver.rs file.

writing 'ScriptThread.add_mutation_observer(observer)' in the code, gives me an error(as expected),  that ScriptThread is "not found in this scope", (probably because I havent inported anything for ScriptThread in the mutationobserver.rs file yet...) but I am not sure what exactly do I need to import in order to access the Script Thread and its methods.